server/services/supportedPrisonsService.ts

```diff
@@ -10,6 +10,10 @@ export default class SupportedPrisonsService {
     private readonly systemToken: SystemToken,
   ) {}
 
+  async getSupportedPrison(prisonId: string): Promise<Prison> {
```

Comment:
Doesn't actually need to be `async` at the moment but figured it will be once we have more prisons and move from static file of values to looking them up in API.